<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>continue_game</class>
 <widget class="QDialog" name="continue_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Обнаружена сохранённая игра, продолжить?</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="continue_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font: 8pt &quot;Comic Sans MS&quot;;
border-image: url(:/main/small_button.png);
}
QPushButton:pressed {
font: 8pt &quot;Comic Sans MS&quot;;
border-image: url(:/main/small_button_pressed.png);
}</string>
   </property>
   <property name="text">
    <string>Да</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_game_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font: 8pt &quot;Comic Sans MS&quot;;
border-image: url(:/main/small_button.png);
}
QPushButton:pressed {
font: 8pt &quot;Comic Sans MS&quot;;
border-image: url(:/main/small_button_pressed.png);
}</string>
   </property>
   <property name="text">
    <string>Новая игра</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>98</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font: 8pt &quot;Comic Sans MS&quot;;
border-image: url(:/main/small_button.png);
}
QPushButton:pressed {
font: 8pt &quot;Comic Sans MS&quot;;
border-image: url(:/main/small_button_pressed.png);
}</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>continue_game</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
